practical/matlab/idr0021_steps.m

```diff
@@ -0,0 +1,203 @@
+% Note: Do not run the whole script. Select the code block
+% of each exercise, right-click and "Evaluate Selection". 
+% Then proceed to the next exercise.
+% The exercises build on top of each other (later exercises
+% cannot be run unless the previous exercises have been 
+% executed successfully). If you get stuck, clear the workspace
+% and try again from the beginning.
+
+% Exercise 1
+% Connect to OMERO and print your group ID.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+client = loadOmero('outreach.openmicroscopy.org');
+session = client.createSession('USER', 'PASSWORD');
+client.enableKeepAlive(60);
+eventContext = session.getAdminService().getEventContext();
+groupId = eventContext.groupId;
+disp(groupId);
+
+
+% Exercise 2
+% List the images of a particular dataset.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+datasetId = MATLAB_DATASET_ID;
```

Comment:
``MATLAB_DATASET_ID;`` - I was confused a bit for a while, because in OMERO, the dataset is named ``matlab-dataset``

Reply:
As discussed, only ``DATASET_ID`` for here.